src/Components/BottomNav/BottomNav.css

```diff
@@ -0,0 +1,46 @@
+@import url("https://effortlessui.netlify.app/base.css");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```

Comment:
Hi Rohan,
Importing your library in all the CSS files is redundant.
You can import it once in the styles.css file or in the Index.html file or any other files.
This would make it globally available.